src/Value/Finance/Iban.php

```diff
@@ -0,0 +1,36 @@
+<?php
+declare(strict_types=1);
+
+namespace MyOnlineStore\Common\Domain\Value\Finance;
+
+use IsoCodes\Iban as IbanValidator;
+use MyOnlineStore\Common\Domain\Exception\Finance\InvalidIban;
+
+final class Iban
+{
+    /**
+     * @var string
+     */
+    private $iban;
+
+    public function __construct(string $iban)
```

Comment:
Note to self: Verify all current constructor usages in monolith in part 2 are converted